### PR TITLE
framework/media: Modify audio_decoder_init_decoder function name.

### PR DESCRIPTION
**Pull request by jiawei-xiang**
@Taejun-Kwon
 Issue: In decoder, api name is hard to understand, audio_decoder_init_decoder & audio_decoder_init
 Solution: modify audio_decoder_init_decoder to audio_decoder_cofig_decoder.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [6ac123d36c01cd718b9b8e8ec76d98653af8fe35] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6ac123d36c01cd718b9b8e8ec76d98653af8fe35] - Code Rule Check OK.

File: framework/src/media/Decoder.cpp [hunk 1]
```diff
@@ -135,8 +135,8 @@ bool Decoder::mConfig(int audioType)
 		mp3_ext.pOutputBuffer = outputBuf;
 		mp3_ext.outputFrameSize = sizeof(outputBuf) / sizeof(int16_t);
 
-		if (audio_decoder_init_decoder(&mDecoder, audioType, &mp3_ext) != AUDIO_DECODER_OK) {
-			meddbg("Error! audio_decoder_init_decoder failed!\n");
+		if (audio_decoder_config_decoder(&mDecoder, audioType, &mp3_ext) != AUDIO_DECODER_OK) {
```
Comment (junmin-kim) [inline framework/src/media/Decoder.cpp]: How about `audio_decoder_configure`?
duplicated `decoder` seems not good, and function name should be verb

**Comment by jiawei-xiang** [comment]
@junmin-kim  Thanks for your suggestion, I have fixed it. :)

**Comment by seinfra** [comment]
Target : [a6548fe11f5427f3d1ede0a2d8ef241652d4bca6] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a6548fe11f5427f3d1ede0a2d8ef241652d4bca6] - Code Rule Check OK.

**Comment by jiawei-xiang** [comment]
@Taejun-Kwon  I checked my modification,  and find that my modification has no conflict with #2141. 
And it seems not need update.
Do I have some misunderstanding? :)

**Pull request merged**